Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>// Top-level build file where you can add configuration options common to all sub-projects/modules.\nbuildscript {\n    repositories {\n        google()\n        mavenCentral()\n    }\n    dependencies {\n        classpath \"com.android.tools.build:gradle:7.0.1\"\n        classpath \"org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.30\"\n        //Dagger\n        classpath 'com.google.dagger:hilt-android-gradle-plugin:2.38.1'\n\n        // NOTE: Do not place your application dependencies here; they belong\n        // in the individual module build.gradle files\n    }\n}\n\ntask clean(type: Delete) {\n    delete rootProject.buildDir\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
--- a/build.gradle	(revision ab0fb12c18eda42176d7c3e7abddab4cb124218c)
+++ b/build.gradle	(date 1631563180146)
@@ -5,7 +5,7 @@
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:7.0.1"
+        classpath 'com.android.tools.build:gradle:7.0.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.30"
         //Dagger
         classpath 'com.google.dagger:hilt-android-gradle-plugin:2.38.1'
