Index: app/src/main/java/com/example/book/model/Book.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.book.model\n\nimport com.google.gson.annotations.SerializedName\n\ndata class BookResponse(\n\n    @SerializedName(\"items\")\n    val items : List<Book>\n)\n\ndata class Book(\n\n    @SerializedName(\"id\")\n    val id: String,\n    @SerializedName(\"volumeInfo\")\n    val volumeInfo: VolumeInfo,\n    @SerializedName(\"publishedDate\")\n    val publishedDate: String,\n    @SerializedName(\"description\")\n    val description: String,\n    @SerializedName(\"pageCount\")\n    val pageCount: Int,\n    @SerializedName(\"imageLinks\")\n    val imageLinks: ImageLinks?\n\n)\ndata class VolumeInfo(\n\n    @SerializedName(\"title\")\n    val title: String,\n    @SerializedName(\"authors\")\n    val authors: List<String>\n)\n\ndata class ImageLinks(\n\n    @SerializedName(\"smallThumbnail\")\n    val smallThumbnail: String?,\n    @SerializedName(\"thumbnail\")\n    val thumbnail: String?\n)
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/book/model/Book.kt b/app/src/main/java/com/example/book/model/Book.kt
--- a/app/src/main/java/com/example/book/model/Book.kt	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
+++ b/app/src/main/java/com/example/book/model/Book.kt	(date 1630784765966)
@@ -22,7 +22,6 @@
     val pageCount: Int,
     @SerializedName("imageLinks")
     val imageLinks: ImageLinks?
-
 )
 data class VolumeInfo(
 
Index: .idea/.name
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.name b/.idea/.name
deleted file mode 100644
--- a/.idea/.name	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
+++ /dev/null	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
@@ -1,1 +0,0 @@
-Book+
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
+++ /dev/null	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
@@ -1,3 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
deleted file mode 100644
--- a/.idea/vcs.xml	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
+++ /dev/null	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
-  </component>
-</project>
\ No newline at end of file
Index: .idea/gradle.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/gradle.xml b/.idea/gradle.xml
deleted file mode 100644
--- a/.idea/gradle.xml	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
+++ /dev/null	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
@@ -1,20 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="GradleMigrationSettings" migrationVersion="1" />
-  <component name="GradleSettings">
-    <option name="linkedExternalProjectsSettings">
-      <GradleProjectSettings>
-        <option name="testRunner" value="GRADLE" />
-        <option name="distributionType" value="DEFAULT_WRAPPED" />
-        <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="modules">
-          <set>
-            <option value="$PROJECT_DIR$" />
-            <option value="$PROJECT_DIR$/app" />
-          </set>
-        </option>
-        <option name="resolveModulePerSourceSet" value="false" />
-      </GradleProjectSettings>
-    </option>
-  </component>
-</project>
\ No newline at end of file
Index: .idea/compiler.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/compiler.xml b/.idea/compiler.xml
deleted file mode 100644
--- a/.idea/compiler.xml	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
+++ /dev/null	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="11" />
-  </component>
-</project>
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
deleted file mode 100644
--- a/.idea/misc.xml	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
+++ /dev/null	(revision 4423d926ac3e5d008b17add599614dd3f8fe4b6e)
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="DesignSurface">
-    <option name="filePathToZoomLevelMap">
-      <map>
-        <entry key="app/src/main/res/layout/main_fragment.xml" value="0.1" />
-      </map>
-    </option>
-  </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="11" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/build/classes" />
-  </component>
-  <component name="ProjectType">
-    <option name="id" value="Android" />
-  </component>
-</project>
\ No newline at end of file
